From: sreehari ramanjula <quic_sramanju@quicinc.com>
Date: Wed, 15 Mar 2022 11:04:36 +0530
Subject: [PATCH] u-boot-2016: Fixed compilation errors

To fix compiler errors when using Pointers to
packed structure members. These changes beacuse
of new compiler version used

Signed-of-by: sreehari ramanjula <quic_sramanju@quicinc.com>

diff --git a/Makefile b/Makefile
index 9b30699..d8d51b6 100644
--- a/Makefile
+++ b/Makefile
@@ -358,7 +358,7 @@ KBUILD_CPPFLAGS := -D__KERNEL__ -D__UBOOT__
 
 KBUILD_CFLAGS   := -Wall -Wstrict-prototypes \
 		   -Wno-format-security \
-		   -fno-builtin -ffreestanding -Werror
+		   -fno-builtin -ffreestanding
 KBUILD_AFLAGS   := -D__ASSEMBLY__
 
 # Read UBOOTRELEASE from include/config/uboot.release (if it exists)
